tests/test_rigid_body/test_mesh_rigid_body.py

```diff
@@ -0,0 +1,1340 @@
+                [1.0, 1.0, -1.0, -1.0, 1.0, -1.0, 1.0, 1.0, 1.0, 1.0, 1.0, -1.0],
+            ],
+            [
+                [-1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, 1.0, 1.0, -1.0, -1.0],
+                [1.0, 1.0, -1.0, 1.0, -1.0, 1.0, -1.0, 1.0, 1.0, 1.0, -1.0, -1.0],
+                [-1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, -1.0, -1.0],
+            ],
+            [
+                [-1.0, -1.0, -1.0, -1.0, 1.0, 1.0, 1.0, 1.0, -1.0, -1.0, -1.0, -1.0],
+                [-1.0, -1.0, 1.0, 1.0, 1.0, 1.0, -1.0, -1.0, -1.0, 1.0, -1.0, 1.0],
+                [-1.0, -1.0, 1.0, -1.0, 1.0, 1.0, -1.0, 1.0, 1.0, 1.0, 1.0, 1.0],
+            ],
+        ]
+    )
+
+    face_normals = np.array(
+        [
+            [0, 0, -1.0, -1.0, 0, 0, 1.0, 1.0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0, 0, 1.0, 1.0, -1.0, -1.0],
+            [-1.0, -1.0, 0, 0, 1.0, 1.0, 0, 0, 0, 0, 0, 0],
+        ]
+    )
+    face_centers = np.array(
+        [
+            [
+                -0.333333,
+                0.333333,
+                -1.0,
+                -1.0,
+                0.333333,
+                -0.333333,
+                1.0,
+                1.0,
+                0.333333,
+                -0.333333,
+                0.333333,
+                -0.333333,
+            ],
+            [
+                -0.333333,
+                0.333333,
+                -0.333333,
+                0.333333,
+                -0.333333,
+                0.333333,
+                -0.333333,
+                0.333333,
+                1.0,
+                1.0,
+                -1.0,
+                -1.0,
+            ],
+            [
+                -1.0,
+                -1.0,
+                0.333333,
+                -0.333333,
+                1.0,
+                1.0,
+                -0.333333,
+                0.333333,
+                -0.333333,
+                0.333333,
+                -0.333333,
+                0.333333,
+            ],
+        ]
+    )
+    n_faces = 12
+
+    # setting up test params
+    test_mesh = MockMesh(n_faces, faces, face_centers, face_normals)
+    center_of_mass = np.array([0.0, 0.0, 0.0])
+    direction = np.array([0.0, 0.0, 1.0]).reshape(3, 1)
+    normal = np.array([1.0, 0.0, 0.0]).reshape(3, 1)
+    binormal = np.array([0.0, 1.0, 0.0]).reshape(3, 1)
+    base_length = 2
+    volume = base_length ** 3
+    density = np.random.uniform(1.0, 10)
+    mass = density * volume
+    # Mass second moment of inertia for cube
+    mass_second_moment_of_inertia = np.zeros((3, 3), np.float64)
+    np.fill_diagonal(mass_second_moment_of_inertia, (mass * base_length ** 2) / 6)
+    # Inverse mass second of inertia
+    inv_mass_second_moment_of_inertia = np.linalg.inv(mass_second_moment_of_inertia)
+
+    test_mesh_rigid_body = MeshRigidBody(
+        test_mesh, center_of_mass, mass_second_moment_of_inertia, density, volume
+    )
```

Comment:
This code repeats in every function. Make an initializer function somewhere top in this function and remove all these repeats. It makes harder to read the tests.